Sources/AsyncHTTPClient/AsyncAwait/HTTPClientRequest.swift

```diff
@@ -143,4 +155,19 @@ extension Optional where Wrapped == HTTPClientRequest.Body {
     }
 }
 
+@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+extension HTTPClientRequest.Body {
+    public struct Length {
+        /// size of the request body is not known before starting the request
+        public static let dynamic: Self = .init(storage: .dynamic)
```

Comment:
Is dynamic the right name here? What about `unknown`?
I like the storage workaround btw.

Reply:
works for me

Reply:
btw. fixed is also weird. I guess `known` would be more appropriate. Also I think this type should be `ExpressibleByIntegerLiteral`.